i18n/en.json

```diff
@@ -512,6 +512,24 @@
             },
             "updatedDescription": "The bot's language has been updated to `{{language}}`."
         },
+        "lookup": {
+            "help": {
+                "description": "Retrieve whether the given song is included KMQ's database and the game options, in addition to song data.",
```

Comment:
Check whether a given Youtube video is included in KMQ's database and current game options